OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -179,7 +180,9 @@ public override Activity Tick(Actor self)
 			if (firstFacing != mobile.Facing)
 			{
 				path.Add(nextCell.Value.First);
-				return ActivityUtils.SequenceActivities(new Turn(self, firstFacing), this);
+				var turn_activity = new Turn(self, firstFacing);
+				turn_activity.NextActivity = this;
+				return turn_activity;
```

Comment:
The removed code and the added code should be doing the same thing so why the change?